packages/expo/src/launch/registerRootComponent.web.tsx

```diff
@@ -5,11 +5,10 @@ import withExpoRoot from './withExpoRoot';
 import { InitialProps } from './withExpoRoot.types';
 
 export default function registerRootComponent<P extends InitialProps>(
-  component: React.ComponentClass<P>
+  component: React.ComponentType<P>
 ): void {
   const App = withExpoRoot(component);
-  // @ts-ignore: TypeScript says ComponentClass<P> does not satisfy ComponentClass<any>
-  const RootComponent = (...props) => <App exp={{}} {...props} />;
```

Comment:
is this change correct?

It made `expo-module build` happy in this module.

Reply:
I dunno how `AppRegistry.registerComponent` works, but if it expects to be passed a react component, it's correct 🙂

Reply:
Yeah, I think this works and CI does type checks and passes.